tests/test_session.py

```diff
@@ -13,9 +13,10 @@
 from ansys.api.fluent.v0 import scheme_eval_pb2, scheme_eval_pb2_grpc
 from ansys.api.fluent.v0.scheme_pointer_pb2 import SchemePointer
 import ansys.fluent.core as pyfluent
-from ansys.fluent.core import connect_to_fluent, examples
+from ansys.fluent.core import connect_to_fluent, examples, session
 from ansys.fluent.core.examples import download_file
 from ansys.fluent.core.fluent_connection import FluentConnection
+from ansys.fluent.core.launcher.launcher import LaunchFluentError
```

Comment:
(if accepting my other suggestion, I believe we need to remove this one as well)
```suggestion
```